web/app/controller/Root.js

```diff
@@ -167,6 +167,9 @@ Ext.define('Traccar.controller.Root', {
                 self.updateEvents(data.events);
             }
         };
+        if (first) {
+            this.first = true;
```

Comment:
The problem is that `updatePositions` is called from `socket.onmessage` callback and we need switch it off after first frame.